packages/state-manager/src/sagas/messages/messages.master.saga.ts

```diff
@@ -22,6 +23,7 @@ export function* messagesMasterSaga(socket: Socket): Generator {
     takeEvery(messagesActions.incomingMessages.type, verifyMessagesSaga),
     takeEvery(messagesActions.incomingMessages.type, markUnreadChannelsSaga),
     takeEvery(messagesActions.incomingMessages.type, updateNewestMessageSaga),
+    takeEvery(messagesActions.updateMessagesVerification.type, updateMessagesVerificationSaga),
```

Comment:
Are you sure this is the proper way? As I see `verifyMessagesSaga` is being called on every `publicChannelsActions.cacheMessages` action so whenever we update channel's cache we verify signatures (it happens on incomingMessages and lazyLoading) so what you did seems like a workaround but it multiplies code instead of fixing already existing logic

Reply:
but we verify signature only one time for one message, and in case with broken verification (when app is in background) I'm doing it for non-existing messageVerifyStatus, without it there would never be information about the status of these messages, but agree it's workaround I'm investigating the main cause